underfs/orangefs/pom.xml

```diff
@@ -0,0 +1,51 @@
+<!--
+  ~ The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+  ~ (the “License”). You may not use this work except in compliance with the License, which is
+  ~ available at www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+  ~ either express or implied, as more fully set forth in the License.
+  ~
+  ~ See the NOTICE file distributed with this work for information regarding copyright ownership.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.alluxio</groupId>
+    <artifactId>alluxio-underfs</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>alluxio-underfs-orangefs</artifactId>
+  <name>Alluxio Under File System - OrangeFS</name>
+  <description>Alluxio Under File System implementation for OrangeFS</description>
+
+  <properties>
+    <!-- These need to be defined here as well as in the parent pom so that mvn can run
+         properly from sub-project directories -->
+    <license.header.path>${project.parent.parent.basedir}/build/license/</license.header.path>
+    <checkstyle.path>${project.parent.parent.basedir}/build/checkstyle/</checkstyle.path>
+    <findbugs.path>${project.parent.parent.basedir}/build/findbugs/</findbugs.path>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.orangefs.usrint</groupId>
+      <artifactId>orangefs-jni</artifactId>
```

Comment:
@yupeng9, OrangeFS is using GNU **Lesser** General Public License (**LGPL**) rather than GNU GPL. It means we  may link OrangeFS-JNI as a library without changing Alluxio's Apache License. 

_"The license allows developers and companies to use and integrate software released under the LGPL into their own (even proprietary) software without being required by the terms of a strong copyleft license to release the source code of their own components. "_

If this is still a problem, we can let end-user deploy OrangeFS-JNI in their systems and configure Alluxio CLASSPATH to reach this library. By this way, we might be able to avoid ship OrangeFS related libraries in Alluxio distribution.